hdkeychain/extendedkey_test.go

```diff
@@ -610,6 +616,20 @@ func TestExtendedKeyAPI(t *testing.T) {
 			continue
 		}
 
+		chainCode := key.ChainCode()
+		if !bytes.Equal(chainCode, test.chainCode) {
+			t.Errorf("ChainCode #%d (%s): want %v, got %v", i,
+				test.name, chainCode, test.chainCode)
```

Comment:
Same as with `ChildNum()`.

```suggestion
				test.name, test.chainCode, chainCode)
```